### PR TITLE
Dgraph ingest script

### PR DESCRIPTION
**Pull request by charlie-wasp**
**N.B.** this PR depends on the massive #63, so I set `chore/dgraph` branch as a base for easier review

Usage example:

`yarn ingest --startSeq=11283374 --endSeq=11283615`

Both `startSeq` and `endSeq` args can be omitted

### CODE REVIEW

File: dgraph/ingest.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
+import parseArgv from "minimist";
+import { DgraphIngestor as Ingestor } from "../src/ingest";
+import { Cursor, ICursorResult } from "../src/ingest/cursor";
+import { Connection } from "../src/storage";
+import logger from "../src/util/logger";
+import { DGRAPH_URL } from "../src/util/secrets";
+import { setNetwork as setStellarNetwork } from "../src/util/stellar";
+
+if (!DGRAPH_URL) {
+  logger.error("No dgraph url");
```
Comment (gzigzigzeo) [inline dgraph/ingest.ts]: I believe, we could add instruction like "provide DGRAPH_URL env variable"

File: dgraph/ingest.ts [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+setStellarNetwork().then((network: string) => {
+  logger.info(`Using ${network}`);
+
+  c.migrate()
+    .then(async () => {
+      Cursor.build(startSeq || -1).then(async cursor => {
+        let data: ICursorResult | null;
+
+        while ((data = await cursor.nextLedger())) {
```
Comment (gzigzigzeo) [inline dgraph/ingest.ts]: Why ((?

Comment (charlie-wasp) [reply dgraph/ingest.ts]: Linter added it during autofix. However, linter doesn't like the assignment in condition itself, but I think, it perfectly fits here. I'll remove additional brackets

File: dgraph/ingest.ts (hunk 1 above)
Comment (charlie-wasp) [reply dgraph/ingest.ts]: Yeah, of course, all these log messages were written in a haste :)

File: src/ingest/dgraph.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import { Connection, Store } from "../storage";
+import { LedgerHeader, Transaction } from "../model";
+
+export class DgraphIngestor {
```
Comment (gzigzigzeo) [inline src/ingest/dgraph.ts]: I have doubts about naming here. Word "ingest" is about getting data out of. This class is more about store into.

Comment (charlie-wasp) [reply src/ingest/dgraph.ts]: Is `DgraphImporter` fits better?

**Comment by charlie-wasp** [comment]
I messed up branches, so I have to close this PR and move commits manually to master 😞 No changes are lost though

**Pull request closed without merging**